### ISSUE TITLE
Mojang/Microsoft login

### ISSUE THREAD

**Issue by WesleyBryie** [LAX1DUDE/eaglercraft#47]
Would it be possible to implement premium minecraft account auth into eaglercraft?

**Comment by sussygamedeveloper** [comment]
Just curious, why would you want this?

**Comment by ayunami2000** [comment]
This could be possible, but it would take a lot of implementing of utilities and features removed to reduce size and increase speed. My fork has encryption which allows joining regular offline mode 1.5.2 servers (with a tcp proxy), but I have not yet implemented account authentication. (This is pretty much halfway there.) It isn't a priority for me rn, but it would be a cool thing if I do get it to work. Once I get time and get other stuff off my plate I'll come back to this.

**Comment by WesleyBryie** [comment]
> Just curious, why would you want this?

I would like my server to be in online mode.

**Comment by sussygamedeveloper** [comment]
Makes sense. @ayunami2000 your fork is very useful for playing on normal vanilla servers.
